routes/index/index.js

```diff
@@ -1,26 +1,44 @@
-var configuration = require('../../lib/configuration');
-var express = require('express');
-var server = module.exports = express();
-var origin = require('../../');
-var app = origin();
+const configuration = require('../../lib/configuration');
+const express = require('express');
+const server = module.exports = express();
+const origin = require('../../');
+const app = origin();
+const installHelper = require('../../lib/installHelpers');
+const logger = require('../../lib/logger');
+
+let _versions = {};
 
 server.set('views', __dirname);
 server.set('view engine', 'hbs');
 
-server.get('/', function (req, res, next) {
-  var dateStamp = new Date();
-  var dateStampAsString = dateStamp.toISOString().replace(/[^0-9]/g, '');
-  var isProduction = configuration.getConfig('isProduction');
-	var useAnalytics = configuration.getConfig('useAnalytics') || false;
-  var trackingId = configuration.getConfig('trackingId') || 'UA-XXXXX-Y';
-     
-  res.render('index', {
-  	isProduction: isProduction, 
-  	dateStampAsString: dateStampAsString,
+server.get('/', async function (req, res, next) {
+  const dateStamp = new Date();
+  const dateStampAsString = dateStamp.toISOString().replace(/[^0-9]/g, '');
+  const isProduction = configuration.getConfig('isProduction');
+  const useAnalytics = configuration.getConfig('useAnalytics') || false;
+  const trackingId = configuration.getConfig('trackingId') || 'UA-XXXXX-Y';
+  const versions = await getVersions();
+
+  res.render('index', Object.assign({
+    isProduction: isProduction,
+    dateStampAsString: dateStampAsString,
     loading: app.polyglot.t('app.loading'),
     productName: app.polyglot.t('app.productname'),
     useAnalytics: useAnalytics,
     trackingId: trackingId
-  });
+  }, versions));
+});
 
-});
\ No newline at end of file
+function getVersions(cb) {
```

Comment:
Putting the `async` keyword in front of the function would mean you won't have to explicitly return a promise – you'd only need to return `_versions` which would be wrapped automatically.